File: Lab2/Sharp/Lab2/Lab2/Furniture.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab2
{
    internal class Furniture
    {
        public Furniture(string name, string type, int amount, int price, Color color)
        {
            Name = name;
            Type = type;
            Amount = amount;
            Price = price;
            Color = color;
        }
        public string Name { get; private set; }
        public string Type { get; private set; }
        public Color Color { get; private set; }
        public int Price { get; private set; }
        public int Amount { get; private set; }

        public override string ToString()
        {
            return $"Title : {Name} {Type}\n" +
                   $"Amount: {Amount}\n" +
                   $"Price : {Price}\n" +
                   $"Color : {Color}";
        }
    }
    enum Color
    {
        Red,
        Green,
        Blue,
        Yellow,
        Orange,
        Purple,
        Brown,
        White,
        Black
    }
}

File: Lab2/Python/furniture.py
class Furniture:
    def __init__(self, name: str, kind: str, amount: int, price: int, color: str) -> object:
        self.name = name
        self.kind = kind
        self.amount = amount
        self.price = price
        self.color = color

    def print(self) -> object:
        print(f'Title            : {self.name} {self.kind}\n'
              f'Amount in storage: {self.amount}\n'
              f'Price per one    : {self.price}\n'
              f'Color            : {self.color}')